parasolr/query/tests/test_queryset.py

```diff
@@ -736,3 +736,23 @@ def test_get_item(self):
 
         with pytest.raises(AssertionError):
             assert sqs[:-1]
+
+
+class TestEmptySolrQuerySet:
+
+    def test_init(self):
+        # Can't be instantiated
+        with pytest.raises(TypeError):
+            EmptySolrQuerySet()
+
+    def test_is_instance(self):
+        mocksolr = Mock(spec=SolrClient)
+        sqs = SolrQuerySet(mocksolr)
+        results = Mock()
+        results.docs = ParasolrDict() # empty response
+        sqs.solr.query = Mock(return_value=results)
```

Comment:
You don't need the extra `Mock` calls here; right after you define `mocksolr` you can set it like this:
`mocksolr.query.return_value.docs =  ParasolrDict()`